BackEndFlask/controller/Routes/User_routes.py

```diff
@@ -152,6 +154,11 @@ def add_user():
                 print(f"[User_routes /user?course_id=<int:id> POST] An error occurred enrolling newly created user in course_id: {course_id}, ", user_course)
                 createBadResponse(f"An error occurred enrolling newly created user in course_id: {course_id}!", user_course, "users")
                 return response
+            one_user = makeAdmin(new_user.user_id, request.json["role_id"])
```

Comment:
This logic making an admin isn't clear to me, and I think could be better structured. The makeAdmim function does not always actually make id, it only does when a specific role_id is given. I think the function should always make an admin if called. Additionally, I think the function may be unneeded, at least in the creation case since we can just modify the create_user function.

Reply:
Also, I think Admin/SuperAdmin should be removed from the role table since they do not describe the relationship of a person to a course.

Reply:
I have updated makeAdmin() to take a user_id and make the isAdmin attribute of that user to True. Therefore, makeAdmin() will always make a user, given a valid user_id, an admin! I have also updated create_user() to set isAdmin to true if a role_id is passed in the user_data json and that “role_id” key has a value of 3!

Reply:
I will remove Researcher and SuperAdmin roles and update the rest of the codebase!

Reply:
I will also remove Admin from the rest of the codebase!

Reply:
I will make a new branch for removing Researcher, SuperAdmin, and Admin from the role table and update the rest of the codebase!